lineages/A.2.yml

```diff
@@ -0,0 +1,12 @@
+# yaml-language-server: $schema=../schemas/lineage_schema.yml
+name: B.1.1
```

Comment:
This is the only one whose name doesn't match it's filename.

Reply:
Good catch Angie - looks like a copy/paste bug probably! @corneliusroemer

Reply:
Oops - sorry, thanks for the catch.

In the process of writing some CI scripts that would catch this in the future.